pandas/tests/plotting/test_frame.py

```diff
@@ -3350,6 +3350,66 @@ def test_colors_of_columns_with_same_name(self):
         for legend, line in zip(result.get_legend().legendHandles, result.lines):
             assert legend.get_color() == line.get_color()
 
+    @pytest.mark.parametrize(
+        "index_name, old_xlabel, new_xlabel, old_ylabel, new_ylabel",
+        [
+            (None, "", "new_x", "", ""),
```

Comment:
Not a huge issue but instead of parametrizing xlabel / ylabel as individual arguments can you not just pass one set of labels and maybe have another parametrization on `transpose` to test the other axis? Would probably help readability

Reply:
you are right, actually we can also remove the need of `transpose`, and simply use `old_label` and `new_label` instead. As commented in the PR, for default, `ylabel` should always be empty string, while `xlabel` be the index name. And if assigned, they will be overriden by assigned value.

Pls let me know your thoughts on the simplified tests! thanks! @WillAyd